src/FSLibrary/StellarMission.fs

```diff
@@ -45,6 +48,8 @@ let allMissions : Map<string, Mission> =
                  ("ComplexTopology", complexTopology)
                  ("LoadGeneration", loadGeneration)
                  ("LoadGenerationWithSpikes", loadGenerationWithSpikes)
+                 ("LoadGenerationWithTxSetLimit", loadGenerationWithTxSetLimit)
```

Comment:
Did you also want to add these new missions to the CI?

Reply:
yeah, wanted to make sure you're aware that another change is needed elsewhere

Reply:
Sounds good, I'll keep this in mind while it's blocked by the core PR release.

Reply:
heads up, we can update the CI now